javaparser-core-testing/src/test/java/com/github/javaparser/ast/body/AnnotationDeclarationTest.java

```diff
@@ -0,0 +1,12 @@
+package com.github.javaparser.ast.body;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class AnnotationDeclarationTest {
+    @Test
+    void cantAddField() {
+        assertThrows(IllegalStateException.class, () -> new AnnotationDeclaration().addField(Object.class, "oo"));
+    }
+}
```

Comment:
nitpick: new line at the end of the file
```suggestion
}

```